packages/lxd/src/stop.coffee.md

```diff
@@ -30,6 +36,11 @@ console.info(`The container was stopped: ${$status}`)
         lxc stop #{config.container}
         """
         code_skipped: 42
+      if config.wait
+        await @execute.wait
+          $shy: true
+          command: "lxc info #{config.container} | grep 'Status: Stopped'"
```

Comment:
You shall honor the `retry` and `interval` properties of wait, probably with a different default for retry (eg `3`) to not wait indefinitely. The schema must reflect them as well.